src/settings.rs

```diff
@@ -1,11 +1,27 @@
 use std::path::PathBuf;
+use serde::{Serialize, Deserialize};
+
+extern crate lazy_static; // Required to use macro - https://www.reddit.com/r/rust/comments/a31e8v/whywhy_not_extern_crate/
```

Comment:
I think `use lazy_static::lazy_static;` should work as well according to https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html